lib/services/ios-project-service.ts

```diff
@@ -672,7 +672,7 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 	}
 
 	public async processConfigurationFilesFromAppResources(release: boolean, projectData: IProjectData): Promise<void> {
-		await this.mergeInfoPlists(projectData);
+		await this.mergeInfoPlists({release}, projectData);
```

Comment:
Could you format this (either put spaces around the curly brackets or use VS Code's formatting)